### ISSUE TITLE
no 'use_global_stats' in gluon.nn.BatchNorm()

### ISSUE THREAD

**Issue by tornadomeet** [apache/incubator-mxnet#9419]
hello,

 `use_global_stats` lie in both `ndarray` and `symbol`, but not in `gluon.BatchNorm()`, should we add it in gluon api?

 `use_global_stats` is used when fixed some network layer which use bn during fine-tune.

**Comment by Roshrini** [comment]
@sandeep-krishnamurthy Can you add labels
- Feature Request, Gluon
and close the issue as the PR is already merged.
